### PR TITLE
README: break down specific sections into their own files

### PR DESCRIPTION
**Pull request by AurelienGasser**
- Linux user namespaces
- PyCharm

### CODE REVIEW

File: linux-userns-guide.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+[< Back to README](./README)
+
+Linux user namespaces guide
+---------------------------
+
+This guide details how to enable user namespaces on Linux. This is a required step for running the substra backend.
+
+On Linux systems, all the docker instances create files with `root` permissions.
+TO working correctly in a dev environment, we need the files created by our dockers to have the same rights as the ones we use to launch our celery tasks.
```
Comment (GuillaumeCisco) [inline linux-userns-guide.md]: Beware `TO working`

File: linux-userns-guide.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+```bash
+guillaume:999:1
+guillaume:165536:65536
+```
+
+The first line should be added with the docker group (999 in my case).
+
+The final step is to re-download all the docker images. Go to the [hlf-k8s](https://github.com/SubstraFoundation/hlf-k8s) project and rerun the `./bootstrap.sh` script.
+Do not forget to build the substra-model image as described in the step 9 of the [README](./README).
```
Comment (GuillaumeCisco) [inline linux-userns-guide.md]: This last part is not needed anymore

Comment (AurelienGasser) [reply linux-userns-guide.md]: @GuillaumeCisco only the last line "Do not forget ..." ?

**Comment by AurelienGasser** [comment]
@GuillaumeCisco I moved the two new `.md` files to a `doc/` folder, along with the pycharm screenshots: https://github.com/SubstraFoundation/substra-backend/pull/39/commits/c6b0391ee72be40103eccea84b19728826f08b73

File: linux-userns-guide.md (hunk 1 above)
Comment (AurelienGasser) [reply linux-userns-guide.md]: Fixed https://github.com/SubstraFoundation/substra-backend/pull/39/commits/f22e2beb456b1576ddbe2fb630462c486c3124a9

File: linux-userns-guide.md (hunk 2 above)
Comment (GuillaumeCisco) [reply linux-userns-guide.md]: Yes

Comment (AurelienGasser) [reply linux-userns-guide.md]: Fixed https://github.com/SubstraFoundation/substra-backend/pull/39/commits/96a12f3f82c4c94c5c0467f68194a30419f67a93

**Pull request merged**